packages/server/utils/AtlassianServerManager.ts

```diff
@@ -66,9 +66,9 @@ class AtlassianServerManager extends AtlassianManager {
     return new AtlassianServerManager(accessToken, refreshToken)
   }
 
-  refreshToken?: string
+  refreshToken: string
 
-  constructor(accessToken: string, refreshToken?: string) {
+  constructor(accessToken: string, refreshToken: string) {
```

Comment:
let's completely remove `refreshToken` as a field on the Manager. 
big picture, the spirit of a refreshToken is something that is kept super secure, seldom used, and never passed around. 

Changes:
- `manager.fetchToken` should return `{accessToken, refreshToken, error, scope}`

then, if you need a new instance like in `addAtlassianAuth` it just looks like this:
```ts
 const {accessToken, refreshToken} = await AtlassianServerManager.init(code)
 const manager = new AtlassianServerManager(accessToken)
```

doing this means the manager never needs a refresh token in the constructor 🎉

Reply:
This is a very good idea, the only question remaining is do we want to move error handling outside of the `fetchToken` method? You mentioned `manager.fetchToken should return {accessToken, refreshToken, error, scope}` and it'd mean we'd need to remove this check from `fetchToken` 
```
    if (error) {
      throw new Error(`Atlassian: ${error}`)
    }
``` 
and check for error whenever we use it manually. I think it's better that way as we can handle it without try/catch, just wanted to make sure if that's what you meant

Reply:
I've removed throwing and I check the error manually, IMO that's cleaner but let me know if you think it's better to do it differently, we can discuss 😁

Reply:
since we never put that in a try/catch, we didn't do any error checking anyways. now we can & we can do it without the try/catch block so it should be cleaner